lib/utils/errors.js

```diff
@@ -23,6 +23,22 @@ class DecryptError extends EvervaultError {}
 
 class RelayOutboundConfigError extends EvervaultError {}
 
+class UnauthorizedError extends EvervaultError {}
+
+class ForbiddenError extends EvervaultError {}
```

Comment:
Not sure all those errors are necessary, especially if they extend `EvervaultError`.
I'd find it cleaner to just declare different errors for things users might want to catch to implement retry logic i.e. `FunctionTimeoutError`, `FunctionNotReadyError`, `FunctionRuntimeError`, etc.

Reply:
Per offline, these changes will be made in a subsequent PR